crates/biome_js_analyze/src/lint/suspicious/no_focused_tests.rs

```diff
@@ -115,27 +161,63 @@ impl Rule for NoFocusedTests {
 
         let mut mutation = ctx.root().begin();
 
+        // Special handling for test.only.each() pattern
+        let callee_str = callee.to_string();
+        if callee_str.contains(format!(".{ONLY_KEYWORD}.each").as_str()) {
+            if let Some(static_member) = callee.as_js_static_member_expression() {
+                if let Ok(obj) = static_member.object() {
+                    if let Some(parent) = obj.as_js_static_member_expression() {
+                        if let Ok(member) = parent.member() {
+                            if member.to_string() == ONLY_KEYWORD {
```

Comment:
Same thing here, let's avoid nested ifs with a more functional approach, and let's avoid the allocation of a string